util/kustomize/testdata/kustomization_yaml_components/components/deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-deployment
+  labels:
+    app: nginx
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - name: nginx
+        image: nginx:1.15.4
```

Comment:
any particular reason for version 1.15.4? In `util/kustomize/kustomize_test.go` line 50, version nginx:1.15.5 is used by another test. I think it will be good to use the same version in the same test file, though various versions of nginx do exist across the code base.

Reply:
this is not so important @chengfang , version just used for verify Build result, we are not executing this container anyway, also this deployment.yaml was copied from another test example

Reply:
I guess my concern is, will it cause the test case to download 2 versions of nginx images, which could be waste and slow down the testing?

Reply:
no, it will not, you are not downloading images during this type of tests

Reply:
It is just text comparation